docs/requirements-docs.txt

```
@@ -1,20 +1,14 @@
 numpydoc
-sphinx>=4.0.0
-dask-sphinx-theme>=3.0.0
+sphinx>=8.0.0
+git+https://github.com/dask/dask-sphinx-theme.git@dst-4
```

We need to remove this in favour of a new release before merging

```suggestion
dask-sphinx-theme>=4.0.0
```